kubernetes-azure-python/__main__.py

```diff
@@ -0,0 +1,120 @@
+"""An Azure RM Python Pulumi program"""
+
+import base64
+import pulumi
+from pulumi_azure_native import resources
+from pulumi_azure_native import network
+from pulumi_azure_native import containerservice
+
+# Get some values from the Pulumi configuration (or use defaults)
+azure_cfg = pulumi.Config("azure-native")
+azureLocation = azure_cfg.require("location")
+proj_cfg = pulumi.Config()
```

Comment:
Any reason to have a duplicate assignment here with lines 10 and 12? I'm guessing this is a namespacing thing, but I think you probably want a comment explaining that so people don't immediately remove this line and change the `proj_cfg` to `azure_cfg` and break things.

Reply:
You are correct that it is due to configuration namespacing. I'll add a comment to that effect (and in other languages where a similar mechanism is needed).

Reply:
Resolved via a comment added in c4dea65